database.yml

```diff
@@ -2,20 +2,20 @@ mysql:
   dialect: "mysql"
   database: "pop_test"
   host: {{ envOr "MYSQL_HOST" "127.0.0.1"  }}
-  port: {{ envOr "MYSQL_PORT" "3306"  }}
+  port: {{ envOr "MYSQL_PORT" "3307"  }}
```

Comment:
to prevent port conflict with locally installed versions. it will make it developers easy to test Fizz without having to worry about their existing local version.